### PR TITLE
Update some PHPDOC

### PR DESCRIPTION
**Pull request by jbourdin**
* add @method in Client for autocomplete with __call
* fix some @return in Message

### CODE REVIEW

File: src/Client.php [hunk 1]
```diff
@@ -5,6 +5,18 @@
...
+ *
+ * @method Message attach($attachment)
+ * @method Message clearAttachments()
+ * @method Message disableMarkdown()
+ * @method Message enableMarkdown()
+ * @method Message from(string $username)
+ * @method Message to(string $username)
+ * @method Message withIcon(string $icon)
+ * @method Message send(string $text = null)
+ */
```
Comment (bkrukowski) [inline src/Client.php]: +1

I'd like to add all possibilities:

```
/**
 * @method string getText()
 * @method Message setText(string $text)
 * @method string getChannel()
 * @method Message setChannel(string $channel)
 * @method string getUsername()
 * @method Message setUsername(string $username)
 * @method string getIcon()
 * @method Message setIcon(string $icon)
 * @method string getIconType()
 * @method Message enableMarkdown()
 * @method Message disableMarkdown()
 * @method Message from(string $username)
 * @method Message to(string $channel)
 * @method Message withIcon(string $icon)
 * @method Message attach(mixed $attachment)
 * @method array getAttachments()
 * @method Message setAttachments(array $attachments)
 * @method Message clearAttachments()
 * @method Message send(string $text = null)
 */
```

Comment (OskarStark) [reply src/Client.php]: can you please add them @jbourdin ?

Comment (jbourdin) [reply src/Client.php]: done

**Comment by jbourdin** [comment]
outdated and probably irrelevent

**Pull request closed without merging**